osbs/api.py

```diff
@@ -516,7 +516,11 @@ def _create_build_config_and_build(self, build_request):
 
     def _check_labels(self, repo_info):
         df_parser = repo_info.dockerfile_parser
-        labels = utils.Labels(df_parser.labels)
+        # DockerfileParse does not ensure a Dockerfile exists during initialization
+        try:
+            labels = utils.Labels(df_parser.labels)
+        except IOError:
+            raise IOError('Could not parse Dockerfile in %s' % df_parser.dockerfile_path)
```

Comment:
I'd like to see reason of failure in log entry (not found, permission, etc...)

Is a good idea to reuse IOError? It is stdlib exception and somehow violates it usage (i.e. by not setting the errno and other OS related stuff) 
```
except IOError as e:
    raise RuntimeError('Could not parse Dockerfile in {}: {}'.format(df_parser.dockerfile_path, e))
```